redfish-core/lib/log_services.hpp

```diff
@@ -1618,6 +1624,40 @@ inline void updateProperty(const crow::Request& req,
             dbus::utility::DbusVariantType(*resolved));
         BMCWEB_LOG_DEBUG << "Set Resolved";
     }
+#ifdef BMCWEB_ENABLE_IBM_MANAGEMENT_CONSOLE
+    if (oemObject)
+    {
+        std::optional<nlohmann::json> bmcOem;
+        if (!json_util::readJson(*oemObject, asyncResp->res, "OpenBMC", bmcOem))
+        {
+            return;
+        }
+        if (!json_util::readJson(*bmcOem, asyncResp->res, "ManagementSystemAck",
+                                 managementSystemAck))
+        {
+            BMCWEB_LOG_ERROR << "Could not read managementSystemAck";
+            return;
+        }
+    }
+    if (managementSystemAck.has_value())
```

Comment:
Don't need .has_value() but it doesn't hurt anything 

E.g. to check myself wrote quick code at https://cpp.sh/
```
#include <iostream>
#include <optional>

int main()
{
    
    std::optional<bool> managementSystemAck;
    if (managementSystemAck)
    {
      std::cout<<"has a value\n";
    }
    else
    {
      std::cout<<"no value\n";
    }
    managementSystemAck = false;
    if (managementSystemAck)
    {
      std::cout<<"Take 2: has a value\n";
    }
    else
    {
      std::cout<<"Take 2: no value\n";
    }    
    
    return 0;
}

```


no value
Take 2: has a value